liquidity/liquidity.go

```diff
@@ -1052,7 +1052,8 @@ func (m *Manager) checkExistingAutoLoops(ctx context.Context,
 	var summary existingAutoLoopSummary
 
 	for _, out := range loopOuts {
-		if out.Contract.Label != labels.AutoloopLabel(swap.TypeOut) {
+		if out.Contract.Label != labels.AutoloopLabel(swap.TypeOut) &&
+			out.Contract.Label != labels.EasyAutoloopLabel(swap.TypeOut) {
```

Comment:
nit: line length

maybe do something like 
```
label := out.Contract.Label
outLabel := labels.AutoloopLabel(swap.TypeOut)
easyOutLabel := labels.EasyAutoloopLabel(swap.TypeOut)
if label != outLabel && label != easyOutLabel {
	continue
}
```

Reply:
I created a helper, as we might get even more labels in the future, so we can isolate all checks in there